### ISSUE TITLE
Channel list always show unread bold text

### ISSUE THREAD

**Issue by pixnbit** [GetStream/stream-chat-android#589]
Even the sample app in the repo has this bold status never changed. 4.11-beta7
![out](https://user-images.githubusercontent.com/729730/89570385-fca90880-d7da-11ea-9456-3bddc79815ab.gif)

This is the propty for the list view:

```xml
    app:streamReadStateAvatarHeight="15dp"
    app:streamReadStateAvatarWidth="15dp"
    app:streamReadStateTextSize="10sp"
    app:streamShowReadState="true"
```

**Comment by adasiewiczr** [comment]
Hi @pixnbit thanks for reporting that. It should be already fixed in the latest version (4.3.0-beta-1).  Let us know if this issue still exists

**Comment by pixnbit** [comment]
Yea, it's fixed, thanks!
